src/async/campaigns.js

```diff
@@ -43,6 +43,33 @@ export async function startCampaign(name, congId, publisherId) {
   return results.insertId;
 }
 
+export async function updateCampaign(campaignId, name, start_date, end_date) {
```

Comment:
forJS, let's try to keep variable names camelScript.
`startDate` instead of `start_date`
`endDate` instead of `end_date`

Reply:
ok. done